zcash_primitives/Cargo.toml

```diff
@@ -25,7 +25,7 @@ byteorder = "1"
 crypto_api_chachapoly = "0.4"
 equihash = { version = "0.1", path = "../components/equihash" }
 ff = "0.8"
-fpe = "0.6"
+fpe = "0.5"
```

Comment:
Could you please provide more info on why is this required?

Reply:
same as above